packages/ckeditor5-dev-tests/bin/check-dependencies.js

```diff
@@ -155,3 +165,34 @@ function groupMissingPackages( missingPackages, currentPackage ) {
 
 	return { dependencies, devDependencies };
 }
+
+/**
+ * Checks whether all packages that have been imported by the CSS file are defined in `package.json` as `dependencies`.
+ * Returned array contains list of missing packages.
+ *
+ * @param {String} fileContent Content of the checking file.
+ * @param {String} filePath An absolute path to the checking file.
+ * @param {Array.<String>>} dependencies Merged list of dependencies and devDependencies.
+ * @returns {Array.<String>|undefined}
+ */
+function parsePostCSS( fileContent, filePath, dependencies ) {
+	const matchedImports = fileContent.match( /^@import "(@ckeditor\/[^/]+)[^"]+";/mg );
```

Comment:
Why only `@ckeditor/*` packages?

Reply:
Now it checks everything.

Reply:
It means – relative file imports, scoped and non-scoped packages.